src/pages/test.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+  <Form
+    :errors="api.errors"
+    :form="v$"
+    :is-form-sent="isFormSent"
+    :submit-name="t('cta')"
+    @submit.prevent="submit"
+  >
+    <LayoutPageTitle is-centered :title="t('title')" />
+    <FormInput :title="t('categories')" type="checkbox">
+      <div class="flex gap-4">
+        <FormCheckbox
```

Comment:
For another display style we'd have to follow the [Inclusively Hiding & Styling Checkboxes and Radio Buttons](https://www.sarasoueidan.com/blog/inclusively-hiding-and-styling-checkboxes-and-radio-buttons/) guide to keep accessibility.